docs/Documentation/Visual-Effects/NPC-Effects/NPC-Hiding.md

```diff
@@ -18,34 +18,38 @@ hide_npcs:
 Where NpcID is declared when you register the NPC which is described [here](../../Features/Npcs.md#provided-integrations).
 ## Full examples
```

Comment:
Just realised that we now have a heading here too for the example, would be nice if you reave that too now.